devices/src/legacy/serial.rs

```diff
@@ -50,14 +50,16 @@ const DEFAULT_MODEM_CONTROL: u8 = 0x8; // Auxiliary output 2
 const DEFAULT_MODEM_STATUS: u8 = 0x20 | 0x10 | 0x80; // data ready, clear to send, carrier detect
 const DEFAULT_BAUD_DIVISOR: u16 = 12; // 9600 bps
 
+pub type SerialInterrupt = Box<Fn() -> result::Result<(), std::io::Error> + Send>;
```

Comment:
I'm think this should be generic and be moved to some other place like vm-device.

Reply:
Yes I can move this to `devices/src/lib.rs`